frontend/src/components/HtmlContent/HtmlContent.scss

```diff
@@ -3,6 +3,10 @@
 // (thus wee cannot rely on Tailwind for such content)
 
 .inner-html-content {
+	word-break: break-word;
+	overflow-wrap: break-word;
+	white-space: pre-wrap;
```

Comment:
Seems like this is breaking the style for our links

without pre-wrap:
![Screenshot at 2023-02-18 21-34-59](https://user-images.githubusercontent.com/61631103/219900558-f7a62c4a-9125-4879-a9b7-c5a8a1ea179b.png)

with pre-wrap:
![Screenshot at 2023-02-18 21-35-11](https://user-images.githubusercontent.com/61631103/219900561-b50f7569-ea6a-4735-97fe-10ed7946efba.png)

is it necessary?

could you please double check? if you want to see this locally just run `yarn database:create-mock`  and you'll find this post in the explore timeline :slightly_smiling_face:

Reply:
This is because we want to make sure that newlines and white space characters entered by the user are displayed as they are.


In this case, I would be tempted to omit the `\t`.

```json
"content": "\n\t\t<p>Fine. I'll use Wildebeest!</p>\n\t\t<p>It does look interesting:\n\t\t\t<a href=\"https://blog.cloudflare.com/welcome-to-wildebeest-the-fediverse-on-cloudflare/\"\n\t\t\t\ttarget=\"_blank\"\n\t\t\t\trel=\"nofollow noopener noreferrer\">\n\t\t\t\t\t<span class=\"invisible\">https://</span>\n\t\t\t\t\t<span class=\"ellipsis\">blog.cloudflare.com/welcome-to</span>\n\t\t\t\t\t<span class=\"invisible\">-wildebeest-the-fediverse-on-cloudflare/</span>\n\t\t\t</a>\n\t\t</p>",
```


I have checked with this text.

```json
"content": "<p>#テスト投稿\n長いURLを投稿してみる\nついでに改行も複数いれてみる\n\n\n良いプログラマになるには | プログラマが知るべき97のこと\n<a href=\"https://xn--97-273ae6a4irb6e2hsoiozc2g4b8082p.com/%E3%82%A8%E3%83%83%E3%82%BB%E3%82%A4/%E8%89%AF%E3%81%84%E3%83%97%E3%83%AD%E3%82%B0%E3%83%A9%E3%83%9E%E3%81%AB%E3%81%AA%E3%82%8B%E3%81%AB%E3%81%AF/\">xn--97-273ae6a4irb6e2hsoiozc2g4b8082p.com/%E3%82%A8%E3%83%83%E3%82%BB%E3%82%A4/%E8%89%AF%E3%81%84%E3%83%97%E3%83%AD%E3%82%B0%E3%83%A9%E3%83%9E%E3%81%AB%E3%81%AA%E3%82%8B%E3%81%AB%E3%81%AF/</a></p>",
```

Reply:
yes, I think you're right, sorry I didn't really know but it does look like you can have spaces in your toots

although.... (sorry for being annoying :sweat:) on mastodon it looks that the pre-wrap is added to the `p` inside the status and not to the status itself:
![Screenshot at 2023-02-19 12-21-36](https://user-images.githubusercontent.com/61631103/219947677-77698a17-6337-47a8-93ba-0b2482629f4c.png)

So I think it would be better to have the `pre-wrap` here instead?
https://github.com/cloudflare/wildebeest/blob/9cc290ecc7b1b1e8fd0b3866aaf74da07c6a1101/frontend/src/components/HtmlContent/HtmlContent.scss#L17-L19

I'm not completely sure to be honest... what do you think?

Reply:
| before | `.inner-html-content` | `.inner-html-content  p` |
| :--: | :--: | :--: |
| ![before](https://user-images.githubusercontent.com/3946829/219951702-35ec4169-cc10-4e43-abe9-b5d365b3224c.png) | ![inner-html-content](https://user-images.githubusercontent.com/3946829/219951714-6087f7d0-11ee-41ef-ba14-a6f64737bf13.png) | ![p](https://user-images.githubusercontent.com/3946829/219951726-c01337a1-3296-42aa-b588-651b883a4ab0.png) |

```scss
:global(p) {
	margin-bottom: theme('spacing.4');
	overflow-wrap: break-word;
	white-space: pre-wrap;
}
```

I think it's good!

- The child element of ".inner-html-content" will have a "p" in it.
- The whitespace I want to display goes inside this "p".

If it's not a problem, I will commit the fix, is that ok?

Reply:
yeah the last one's perfect!!! :heart_eyes::+1:

Reply:
(PS: if you could squash all the commits into a single one that'd be awesome :pray:)

Reply:
Fix is complete, please check back! 🙏